phpBB/phpbb/cache/driver/file.php

```diff
@@ -279,6 +279,7 @@ function _exists($var_name)
 		if ($var_name[0] == '_')
 		{
 			global $phpEx;
+			$var_name = str_replace('/', '_', $var_name);
```

Comment:
I'd prefer to use `-` as replacement, because we use `_`  for our keys. 
Using a different character reduces the problem of potential conflicts.